django_lti_tool_provider/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.conf.urls import patterns, url
+
+from django_lti_tool_provider import views as lti_views
+
+urlpatterns = patterns(
+    '',
+    url(r'', lti_views.LTIView.as_view(), name='lti')
+)
```

Comment:
`patterns()` is deprecated in Django 1.8.  You should use a simple list of `url()` calls instead.